snuba/datasets/entities/events.py

```diff
@@ -0,0 +1,160 @@
+from datetime import timedelta
+from typing import Mapping, Sequence
+
+from snuba import state
+from snuba.clickhouse.translators.snuba.mappers import SubscriptableMapper
+from snuba.clickhouse.translators.snuba.mapping import TranslationMappers
+from snuba.datasets.dataset import Entity
```

Comment:
This seems wrong. Entity is defined in snuba.datasets.entity

Reply:
Not sure how any tests passed with this.